acc_test/data/valid_basic_multi_DC_provisioning.tf

```diff
@@ -0,0 +1,50 @@
+provider "instaclustr" {
+  username     = "%s"
+  api_key      = "%s"
+  api_hostname = "%s"
+}
+
+resource "instaclustr_cluster" "valid" {
+  cluster_name = "testcluster"
+  sla_tier     = "NON_PRODUCTION"
+
+  bundle {
+    bundle = "APACHE_CASSANDRA"
+    version = "apache-cassandra-3.11.8.ic2"
+    options = {
+      auth_n_authz = true
+      use_private_broadcast_rpc_address = false
+      client_encryption = false
+      lucene_enabled = false
+      continuous_backup_enabled = true
+    }
+  }
+
+  data_centres {
```

Comment:
[question] don't these `data_centres` need bundles?

Reply:
I don't think this file is used anywhere?

Reply:
removed & resolved.